zqd/api/api.go

```diff
@@ -72,7 +72,14 @@ type ScannerStats struct {
 	RecordsMatched int64 `json:"records_matched"`
 }
 
+type SpaceDesc struct {
+	ID      string `json:"id"`
+	Name    string `json:"name"`
+	DataDir string `json:"data_dir"`
+}
+
 type SpaceInfo struct {
+	ID            string     `json:"id"`
```

Comment:
Having both SpaceDesc and SpaceInfo feels a little funny, perhaps because when you should expect the API to use one or the other ins't really clear from the names. Would it make sense to drop SpaceDesc, add the DataDir field to SpaceInfo, and then use SpaceInfo everywhere?

Reply:
This is really more about how api responses are rendered in json. When you as the api to list all the spaces for zqd, I would imagine that sending back all the metadata associated with the space would be too much information. `SpaceDesc` should really be embedded in `SpaceInfo` though I'm pretty sure encoding/json would output something undesirable.